duradmin/src/main/webapp/js/spaces-manager.js

```diff
@@ -3238,6 +3234,12 @@ $(function() {
       }
 
       this._loadHistoryPanel(space);
+
+      if (this._isAdmin()) {
+        if (space.primaryStorageProvider && this._isS3(space.storeId) && this._isAdmin() && !this._isSnapshot(this._storeId)) {
```

Comment:
Can't this line just be: `if (this._isS3(space.storeId))`?
- It doesn't really matter if the provider is primary, so long as it is S3
- You're already testing for `_isAdmin()` in the line above, so you know it's true
- By definition, if `_isS3` is true you know `_isSnapshot` will be false, so it doesn't seem necessary to test